.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 npm-debug.log
-tmp
+.DS_Store
```

Comment:
Can you please add `.idea/` to this list, and then remove the .DS_Store and .idea/* files from your PR?